### ISSUE TITLE
Add FSGM_APR_SP

### ISSUE THREAD

**Issue by iCGY96** [fra31/auto-attack#69]
**Paper**: [Amplitude-Phase Recombination: Rethinking Robustness of Convolutional Neural Networks in Frequency Domain](https://arxiv.org/abs/2108.08487)

**Venue**: ICCV 2021

**Dataset and threat model**: CIFAR-10, L-inf, 8/255

**Code**: [code](https://github.com/iCGY96/APR/tree/main/autoattacks)

**Pre-trained model**: [model](https://drive.google.com/file/d/1nu3RhKKT7IPXJCTsvZ0D9mCyNV6-PIM7/view?usp=sharing)

**Log file**: [log](https://github.com/iCGY96/APR/blob/main/autoattacks/logs/log_file.txt)

**Additional data**: no

**Clean and robust accuracy**: 84.30% / 45.70%

**Architecture**: RN-18

**Description of the model/defense**: Motivated by the powerful generalizability of the human, we argue that reducing the dependence on the amplitude spectrum and enhancing the ability to capture phase spectrum can improve the robustness of CNN.

**Comment by fra31** [comment]
Hi,

thanks for the submission! I'd add your model, with the evaluation on the full test set, to [RobustBench](https://robustbench.github.io/) since it is currently up-to-date, if this is fine for you.

**Comment by iCGY96** [comment]
Very Thanks.

**Comment by fra31** [comment]
Added with https://github.com/RobustBench/robustbench/commit/a97f10d6523017f547fb4a181b7933e54c8eb46b. If it looks fine to you, I update the website too.

**Comment by iCGY96** [comment]
Thank you for your evaluation. RobustBench is a very good platform. We are not going to add the current results to RobustBench now. We will submit more results uniformly when we are ready for more models about cifar10-c and cifar100-c.
